helm/sematic-server/values.yaml

```diff
@@ -0,0 +1,104 @@
+auth:
```

Comment:
We had documented all the non-obvious parameters in our main documentation [here](https://docs.sematic.dev/diving-deeper/deploy#configuration-1). There's now a lot more here than makes sense to put in that document. So instead, would you mind adding comments for each configuration and section that explain what it does? We want to make deployment possible for people who are fairly minimally familiar with k8s.

Reply:
Created https://github.com/sematic-ai/sematic/issues/347 to track this.